test/e2e/tests/test_table.py

```diff
@@ -155,3 +267,391 @@ def test_enable_ttl(self, forum_table):
         assert ttl["AttributeName"] == "ForumName"
         ttl_status = ttl["TimeToLiveStatus"]
         assert ttl_status in ("ENABLED", "ENABLING")
+
+    def test_enable_stream_specification(self, forum_table):
+        (ref, res) = forum_table
+
+        table_name = res["spec"]["tableName"]
+
+        # Check DynamoDB Table exists
+        assert self.table_exists(table_name)
+
+        # Get CR latest revision
+        cr = k8s.wait_resource_consumed_by_controller(ref)
+
+        # Disable stream
+        cr["spec"]["streamSpecification"] = {
+            "streamEnabled": False,
+        }
+
+        # Patch k8s resource
+        k8s.patch_custom_resource(ref, cr)
+
+        table.wait_until(
+            table_name,
+            table.stream_specification_matches(False),
+            timeout_seconds=30,
+            interval_seconds=3,
+        )
+
+        # Get CR latest revision
+        cr = k8s.wait_resource_consumed_by_controller(ref)
+
+        # Disable stream
+        cr["spec"]["streamSpecification"] = {
+            "streamEnabled": True,
+            "streamViewType": "NEW_AND_OLD_IMAGES"
+        }
+        # Patch k8s resource
+        k8s.patch_custom_resource(ref, cr)
+
+        table.wait_until(
+            table_name,
+            table.stream_specification_matches(True),
+            timeout_seconds=30,
+            interval_seconds=3,
+        )
+
+    def test_update_billing_mode(self, billing_table):
+        (ref, res) = billing_table
+
+        table_name = res["spec"]["tableName"]
+
+        # Check DynamoDB Table exists
+        assert self.table_exists(table_name)
+
+        # Get CR latest revision
+        cr = k8s.wait_resource_consumed_by_controller(ref)
+
+        # Set billing mode to PROVISIONED
+        cr["spec"] = {
+            "billingMode": "PROVISIONED",
+            "provisionedThroughput": {
+                "readCapacityUnits": 5,
+                "writeCapacityUnits": 5
+            }
+        }
+
+        # Patch k8s resource
+        k8s.patch_custom_resource(ref, cr)
+
+        table.wait_until(
+            table_name,
+            table.billing_mode_matcher("PROVISIONED"),
+            timeout_seconds=30,
+            interval_seconds=3,
+        )
+
```

Comment:
Add some `assert`ions here to validate that the values you set for read/write capacity are correctly set in `ProvisionedThroughput`

Reply:
Are you sure?  [From my read](https://github.com/aws-controllers-k8s/dynamodb-controller/blob/117749ea0727440d258afc913a24e05f69c6432e/test/e2e/table.py#L97), `BillingModeMatcher` only matches on the `BillingModeSummary.BillingMode` field; there is no use of `ProvisionedThroughputMatcher` here.